sdk-core/assembly/as_types.d.ts

```diff
@@ -1,5 +1,5 @@
 /// <reference types="assemblyscript/std/assembly" />
-
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
How much noise would it generate to only disable certain lines where `any` appears, rather than the whole file?

Reply:
It wouldn't be too bad, but the issue is that it would be for any use of `any`. Which is needed some times for TS to play nicely with these global types, particularly the decorators.  This is what the AS types file does, though there it would be hella noisy.